### DISCUSSION TITLE
Putting text anywhere on the page

### DISCUSSION THREAD

**Discussion by Elf3874** [saber-notes/saber#1353] [Ideas]
In the current version it only allows text to be started from the begining.

A nice feature would be to have that text be allowed to be placed everywere.

Not having this doesn't make the app unusable since its mostly for handwriten notes, but would be a since to have.

**Comment by jyo64** [comment]
I second this. Feature to put text anywhere (including changing text size) and images anywhere would be a game changer for me. This is the only reason why I'm still jumping across apps.

**Comment by codeouran** [comment]
I would also like to see this feature added. There are times where being able to add text to any location on a page would be extremely useful, especially if I have already added notes to a page or am working on top of another document.

**Comment by Haselnuss1** [comment]
I would love to use saber. Right now im on Samsung Notes. And well... it works how it should, but its not open source, and I don't like that I don't know what the app/cloud does with my notes. But **I can't use** saber. I take notes as a student. Handwritten but also PDFs. And the point that I can't put text wherever I want destroys saber for me.

Please add this feature. It would be such an addition to the already nice app.
